### ISSUE TITLE
Error "Factory file not found" when initializing Calibration

### ISSUE THREAD

**Issue by kosalk** [cubert-hyperspectral/cuvis.python#36]
Cuvis version: _CUBERT SDK v. 3.3.0 build: 6e3299618350f9e10837a7a39f40e810db3a753c_

An error _"cuvis.cuvis_aux.SDKException: Factory file not found."_ is returned when trying to initialize a Calibration object with `cuvis.Calibration(calib_dir)`.

The calibration directory is expected to contain the file _init.daq_, but oftentimes that file has a suffix in its name (e.g., _init_123.daq_) and can't be found. Renaming _init_123.daq_ to _init.daq_ solved the problem.

**Comment by birkholz-cubert** [comment]
Hello,

i hope i can clear some things up for you:
- The `init_123.daq` files in the Calibration get created when saving the measurements as Legacy `.cu3` files and are needed by those measurement files to be loaded correclty. Alternatively the Session File format could be used `.cu3s` (where those files are stored in the file itself.) 
- When creating a Calibration object from a factory file it is indeed the expected behaviour to only readin a file named `init.daq`.
(You would create a seperarate folder and put the init.daq in there.)
Alternatively you can also create a Calibration Object from a Session File.

So in short: The behavior you are seeing is indeed expected when using the Legacy Measurement format. To not have to worry about this linking between measurement files and the Calibration folder it is best to use the newer Session File format.
